index.html

```diff
@@ -46,6 +46,150 @@ <h3>LET'S CONNECT</h3>
+                        <li>javascript</li>
+                    </ul>
+                    <a class="project-details" href="">See Project</a>
+                </div>
+            </section>
+
+            <section class="works-section">
+                <div class="project">
+                    <img src="/images/snapshoot-portfolio-4.svg" alt="">
+                    <h3>Multi-Post Stories</h3>
+                    <div class="canopy">
+                        <p class="title">CANOPY</p>
+                        <img class="counter" src="images/Counter.png" alt="counter" />
+                        <p class="job">Back End Dev</p>
+                        <img class="counter" src="images/Counter.png" alt="counter">
+                        <p class="date">2015</p>
+                    </div>
+                    <p class="project-description">A daily selection of privately personalized reads; no accounts or
+                        sign-ups required.</p>
+                    <ul class="project-tech">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <a class="project-details" href="">See Project</a>
+                </div>
+            </section>
+
+            <section class="about">
+                <h1>About<br> Myself</h1>
+                <p>
+                    I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <h3 id="h3">LET'S CONNECT</h3>
+                <div class="contact">
+                    <ul id="social-media">
+                        <li class="github"><a href="#"><img src="/images/github.png" alt="github link"></a></li>
+                        <li class="linkedin"><a href="#"><img src="/images/Linkedin_icon.png" alt="linked link"></a>
+                        </li>
+                        <li class="doits"><a href="#"><img src="/images/doits.png" alt="doit"></a></li>
+                        <li class="twitter"><a href="#"><img src="/images/twitter.png" alt="twitter link"></a></li>
+                        <li class="m"><a href="#"><img src="/images/M.png" alt="M"></a></li>
+                    </ul>
+                    <a class="about-details" href="">Get My Resume</a>
+                </div>
+
+                <div class="about-skills">
+                    <div id="programming-lanuages">
+                        <div>
+                            <p>Languages</p>
+                            <img class = "Languages-img" src="images/fleche_dessous.png" alt="selected">
+                        </div>
+                        <ul>
+                            <li>
+                                <img class = "Languages-img" src="images/java.png" alt="javascript">
```

Comment:
- [ ] You are almost there! However, the icons are very small compared to the Figma template:

|Template|Your design|
|---|---|
![Captura de pantalla 2022-04-29 a la(s) 10 14 50 p  m](https://user-images.githubusercontent.com/83717285/166088697-f505fc18-fd92-4ab3-92d2-10ed3e5a9f03.png)|![Captura de pantalla 2022-04-29 a la(s) 10 14 40 p  m](https://user-images.githubusercontent.com/83717285/166088714-8528d90e-4f0a-4369-87aa-056f80383f6f.png)|

Let's try to make it bigger as in the template.